public_html/wp-content/plugins/pattern-directory/includes/pattern-post-type.php

```diff
@@ -143,6 +143,25 @@ function register_post_type_data() {
 			),
 		)
 	);
+
+	register_post_meta(
+		POST_TYPE,
+		'wpop_block_types',
+		array(
+			'type'              => 'string',
+			'description'       => 'A list of block types this pattern supports for transforms.',
+			'single'            => false,
+			'sanitize_callback' => function( $value, $key, $type ) {
+				return preg_replace( '/[^a-z0-9-\/]/', '', $value );
```

Comment:
_(not a blocker)_

Is there not already a function _(from Core or elsewhere)_ to sanitize block names?

If not, it might be good to pull this out into one, but that can always be done later too.

Reply:
Not that I found - core just [runs some regexes](https://core.trac.wordpress.org/browser/trunk/src/wp-includes/class-wp-block-type-registry.php#L60) and flags it as `_doing_it_wrong` if they don't match.

Reply:
I'd rather not worry about it now - if we need to validate/sanitize block names again, I'll make it then.

Reply:
👍🏻